src/main/java/com/datatorrent/netlet/DefaultEventLoop.java

```diff
@@ -513,14 +512,18 @@ public String toString()
 
             });
           }
-        }
-        catch (IOException ie) {
-          l.handleException(ie, DefaultEventLoop.this);
+        } catch (UnresolvedAddressException e) {
+          logger.error("Inet Address {} is not resolvable.", address, e);
```

Comment:
Can it be made a configuration setting of AbstractClient to log as error or debug, since it is anyway a specific implementation of handleException that AbstractClient is choosing to do.